CHANGELOG.md

```diff
@@ -27,13 +27,18 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 - The `processResources` metric was added, which keeps a track of all sorts of
   active resources. It consists of the following gauges:
+
```

Comment:
I didn't add this line, for some reason the commit hook is adding it.

Reply:
probably prettier thinking the newline is... prettier 😀